golden_response/staging/get_landing_page_data/city.json

```diff
@@ -4334,67 +4329,68 @@
-          "2020-09-04": 6220292,
-          "2020-09-05": 6262570,
-          "2020-09-06": 6292600,
-          "2020-09-07": 6317773,
-          "2020-09-08": 6346637,
-          "2020-09-09": 6379950,
-          "2020-09-10": 6417986,
-          "2020-09-11": 6465534,
-          "2020-09-12": 6504683,
-          "2020-09-13": 6538052,
-          "2020-09-14": 6574891,
-          "2020-09-15": 6614112,
-          "2020-09-16": 6653328,
-          "2020-09-17": 6698641,
-          "2020-09-18": 6747516,
-          "2020-09-19": 6789360,
-          "2020-09-20": 6825761,
-          "2020-09-21": 6880635,
-          "2020-09-22": 6917928,
-          "2020-09-23": 6959494,
-          "2020-09-24": 7004672,
-          "2020-09-25": 7059208,
-          "2020-09-26": 7101969,
-          "2020-09-27": 7139301,
-          "2020-09-28": 7176466,
-          "2020-09-29": 7219936
+          "2020-07-31": 4571510,
+          "2020-08-01": 4628329,
+          "2020-08-02": 4679122,
+          "2020-08-03": 4726604,
+          "2020-08-04": 4780151,
+          "2020-08-05": 4833868,
+          "2020-08-06": 4891561,
+          "2020-08-07": 4952718,
+          "2020-08-08": 5007958,
+          "2020-08-09": 5056438,
+          "2020-08-10": 5103611,
+          "2020-08-11": 5156968,
+          "2020-08-12": 5211246,
+          "2020-08-13": 5265307,
+          "2020-08-14": 5324784,
+          "2020-08-15": 5375527,
+          "2020-08-16": 5417664,
+          "2020-08-17": 5455187,
+          "2020-08-18": 5498420,
+          "2020-08-19": 5541433,
+          "2020-08-20": 5587462,
+          "2020-08-21": 5636491,
+          "2020-08-22": 5681517,
+          "2020-08-23": 5713850,
+          "2020-08-24": 5754254,
+          "2020-08-25": 5793437,
+          "2020-08-26": 5838756,
+          "2020-08-27": 5884366,
+          "2020-08-28": 5930930,
+          "2020-08-29": 5975540,
+          "2020-08-30": 6008970,
+          "2020-08-31": 6045455,
+          "2020-09-01": 6089504,
+          "2020-09-02": 6121948,
+          "2020-09-03": 6168342,
+          "2020-09-04": 6220446,
+          "2020-09-05": 6262700,
+          "2020-09-06": 6292699,
+          "2020-09-07": 6317865,
+          "2020-09-08": 6346806,
+          "2020-09-09": 6380137,
+          "2020-09-10": 6418197,
+          "2020-09-11": 6465764,
+          "2020-09-12": 6504868,
+          "2020-09-13": 6538211,
+          "2020-09-14": 6575097,
+          "2020-09-15": 6614314,
+          "2020-09-16": 6653583,
+          "2020-09-17": 6698889,
+          "2020-09-18": 6747772,
+          "2020-09-19": 6789583,
+          "2020-09-20": 6825935,
+          "2020-09-21": 6880888,
+          "2020-09-22": 6918272,
+          "2020-09-23": 6959820,
+          "2020-09-24": 7005014,
+          "2020-09-25": 7059596,
+          "2020-09-26": 7102293,
+          "2020-09-27": 7139591,
+          "2020-09-28": 7176949,
+          "2020-09-29": 7220646,
+          "2020-09-30": 7262630
```

Comment:
Large diff from this NYT commit: https://github.com/nytimes/covid-19-data/commit/00b6634ae898a82ce304a9071fcbf9a2696c43e5#diff-98c6679a282af4962b61eb2a87b66466L194-L254